### DISCUSSION TITLE
Low alert beeping sound when low that is different than low alert alarm

### DISCUSSION THREAD

**Discussion by kleenkanteen** [NightscoutFoundation/xDrip#2734] [General]
When my sugar level goes 3mmol (54 mg/dl) or below, I think 3 is the limit, I get this sound every 5 minutes that is different from my single low alert, which is raised every 15 minutes if I am below 3.5. The weird sound is a few beeps and does so with my speaker forced all the way to full volume. It is not like an alert that keeps on going and I have to dismiss it, rather, for 3 seconds it just beeps on it's own every 5 minutes I am 3 or below. I looked through the settings to find how to turn it off, looked through google, https://xdrip.readthedocs.io/en/latest/use/alarms/, and this github repo to find what setting changes it, but I am unable to find any mention of it. Anyone know how?

edit: The source is the BYOD dexcom app, I turned off all notifications in the app and for the app from phone settings but the urgent low alarm still triggers below 3.1.

**Comment by Navid200** [comment]
Please post a screenshot of your Glucose level alerts list.

Please look in the logs (https://navid200.github.io/xDrip/docs/Logs.html) and show what you see at the time you hear the unknown alert.

**Comment by kleenkanteen** [comment]
Pic attached. Look at 21:09 and 21:04, when it says "Snoozing alert due to volume button press". I pressed the down volume button to lower the sound of the loud 3 second beep. Notice how in both cases of this volume down log it is is preceded by Snoozed until xxx". Look at 20:54 "BG LOW ALERT". The snooze at 20:54 was for the normal low alert which I snoozed. It would re raise 15 minutes later, by then I believe it was fixed so it did not reraise. So it looks like some alert but it's not the normal low alert that you snooze, but just some 3 seconds of beeping.

![blood](https://user-images.githubusercontent.com/67878058/225493212-23ca73da-adcd-47ea-a335-2c0bbf8ee663.jpg)

**Reply by Navid200** [reply]
It may help if you show your list of Glucose level alerts.

**Reply by kleenkanteen** [reply]
yup on it

**Reply by kleenkanteen** [reply]
Here is my glucose levels alert:

![list 1](https://user-images.githubusercontent.com/67878058/225493832-1677326a-3a08-4d63-908a-0d5e8773d240.jpg)

And inside the low alert:

![list 2](https://user-images.githubusercontent.com/67878058/225493856-ef132119-8bb6-44a4-83a7-1eaf9ef3add8.jpg)

**Comment by Navid200** [comment]
What is the hardware data source of xDrip?

**Reply by kleenkanteen** [reply]
640G / Eversense. Technically the modified official Dexcom app, BYOD Dexcom, https://www.reddit.com/r/dexcom/comments/y4vixk/oc_build_your_own_dexcom_app_update_now_with_g7/

**Reply by kleenkanteen** [reply]
I turned off all notifications in the dexcom app

**Reply by Navid200** [reply]
Were you able to snooze it through xDrip?

**Reply by kleenkanteen** [reply]
Snooze the normal alerts? Yes. In the modified dexcom app I disabled all notifications that arise so it shouldn't be coming from there. I also turned off notifications permissions for the dexcom app.

**Reply by Navid200** [reply]
You say you have an unidentified alert sound.

Question:
When you heard it, were you able to snooze it (the unidentified alert) through xDrip?

**Reply by kleenkanteen** [reply]
Nope. No snooze option came up. The graph was on my screen like normal so nothing happened visually.

**Reply by Navid200** [reply]
xDrip does not have a ghost alert.

Based on what you have described. the sound must be coming from an app other than xDrip.
I understand you say you have disabled the BYODA alerts.  Check again.  The next time you hear this particular alert, open BYODA, not xDrip.  Open BYODA and see if it is showing anything.

**Reply by kleenkanteen** [reply]
Will do, thanks for the pointer

**Reply by michaeld427** [reply]
You can't disable the Urgent Low alert in the regular G6 app.  Unless that is a special feature in the BYOD version, that is likely the sound you are getting.

**Reply by kleenkanteen** [reply]
Ah, in BYOD, the urgent low alert sound "Fall Rate" is that very sound. I turned off all notifications for BYOD in both the app and phone settings so odd how the sound still goes through. On second thought, I think since the dexcom app is always on, and the sound is not a notification sound but a media sound so it can sound. That's why my media sound goes to the max when that sound happens.
